src/main/java/mate/Main.java

```diff
@@ -0,0 +1,38 @@
+package mate;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.lib.Injector;
+import mate.model.Movie;
+import mate.model.MovieSession;
+import mate.service.MovieService;
+import mate.service.MovieSessionService;
+
+public class Main {
+    private static final Injector injector = Injector.getInstance("mate");
+
+    public static void main(String[] args) {
+        MovieService movieService = (MovieService)injector
+                .getInstance(MovieService.class);
+        Movie fastAndFurious = new Movie("Fast and Furious");
+        fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
+        movieService.add(fastAndFurious);
+        System.out.println(movieService.get(fastAndFurious.getId()));
+        movieService.getAll().forEach(System.out::println);
+
+        System.out.println("__________________________________");
+        MovieSessionService movieSessionService = (MovieSessionService)injector
+                .getInstance(MovieSessionService.class);
+        MovieSession movieSession = new MovieSession();
+        movieSession.setMovie(fastAndFurious);
+        movieSession.setShowTime(LocalDateTime.now());
+        movieSessionService.add(movieSession);
+        System.out.println(movieSession);
+        System.out.println("____________________________________");
+        System.out.println(movieSessionService.findAvailableSessions(1L, LocalDate.now()));
```

Comment:
Testing is poor here. Code does not work correctly. We should see two sessions in the list
![image](https://user-images.githubusercontent.com/20326858/207849194-5f4c4f74-3334-4691-b4bf-cf0672e79337.png)
![image](https://user-images.githubusercontent.com/20326858/207849303-8e29b19c-1afe-430a-a981-c0ae83cbc9a1.png)